contrib/lang/markdown/packages.el

```diff
@@ -99,3 +100,22 @@ which require an initialization must be listed explicitly in the list.")
 (defun markdown/init-markdown-toc ()
   (use-package markdown-toc
     :defer t))
+
+(defun markdown/init-mmm-mode ()
+  (use-package mmm-mode
+    :config
```

Comment:
I would wrap the whole `:config` block with a `progn` to be consistent with the rest of spacemacs.